new/LIT5868Hymn.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5868Hymn" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Hymn to Demetrius</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="am">Amharic</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2020-03-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="am">
+                    <note>The following incipit is taken from <ref type="mss" corresp="BLorient535"/>.</note>
+                    <ab>መወድስ፡ ዘዘማርያም፡ የስንዴ፡ ዛላ፡ ወሪቅ፡ ድሜጥሮስ፡ ድሜጥሮስ፡ ለጋ፡ የዶግ፡ ድሜጥሮስ፡ ለምለምለም፡ 
+                        ስንደዶ፤ በክሮ፡ ሎሚ፡ ድሜጥሮስ፡ የመጌዶ፤</ab>
+                </div>
```

Comment:
isAbout relation could be helpful (PRS3499Demetriu?) but of course it is up to you to decide

Reply:
የዶግ፡ ድሜጥሮስ፡ ለምለምለም፡ - ??? If you did not mark it - I think "mawaddes" in the beginning is the type of the hymn ("mawaddes written by Za-Maryam"), and the language seems to be Amharic.

Reply:
It is a bit to unclear for me to identify PRS3499Demetriu with certainty...

Reply:
Demetros is the protagonist of thie "mawaddes"/praise. I am not sure that we need here an ID for him, rather for Za-Maryam as the author.